src/main/resources/default-job.json

```diff
@@ -21,6 +21,13 @@
     "libraries": [
       {
         "jar": "s3://${projectProperties['databricks.repo']}/${projectProperties['databricks.repo.key']}"
+      },
+      {
+        "maven": {
```

Comment:
why would this be a default?

I think the confusion here is that you added it for documentation purposes?

Reply:
Oh, sure. It shouldn't be a default lib. I'd better move it to the README.md